apiary.apib

```diff
@@ -1557,6 +1561,14 @@ Retrieves all the different Clearings card types for the current client.
         Filter by date range.
     + shop_id (optional, string)
         Filter by shop id
+    + stablishment_id (optional, string)
+        Filter by stablishment id
+    + priority (optional, integer)
+        Filter by priority
+    + created_at_from (optional, date, `2017-01-01T12:00:00+01:00`)
```

Comment:
I would also add, if posible, that we recommend using `iso8601` format. Users will thank us for being transparent on that